### DISCUSSION TITLE
Thank you for over 12.000 downloads since July 2020!

### DISCUSSION THREAD

**Discussion by NiyaShy** [NiyaShy/XB1ControllerBatteryIndicator#57] [General]
Hey folks,

I occasionally take a look at the download count via Githubs API, and according to it the current "main" version 1.3.1 was downloaded more than 12.000 times since the last minor (language) update in 2020.
![grafik](https://user-images.githubusercontent.com/21028106/119388892-2cbe8b00-bccb-11eb-81a9-a5159d7613cf.png)

This count sadly gets reset every time I upload a freshly built version to add new languages, so since I just did that to add the new Hungarian translation the counter is now at 0 again.

It really means a lot to me that this little project is so popular and that thousands of gamers around the world use it. Thank you all, and I'll do my best to keep this project running as long as my time allows it! ❤️

**Comment by MorbidEden** [comment]
This has been integral for several years, so, again, keep up the amazing work.

Worked for me on an Xbox One controller, and despite Microsoft limitations has worked great with an Xbox Series X controller WITH the newer Microsoft dongle (non-Bluetooth.)

Great work. Keep doing what you're doing. <3

**Reply by NiyaShy** [reply]
Thanks ^^ But I never would dare to take all the praise for myself, after all most of the newer additions came from other people.

So I guess in addition to thanking everyone who downloaded and uses it, I'll have to thank those who joined the project and committed new features and translations.
